Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -262,22 +397,31 @@ namespace AzToolsFramework
         HandleReset();
     }
 
+    void DocumentPropertyEditor::addAfterWidget(QWidget* precursor, QWidget* widgetToAdd)
```

Comment:
nit: Should this be AddAfterWidget?

Reply:
yeah, I just live in Qt world where functions are lower camel case (because it's a better world!!!! FREEEEEDOMMMMMMMM!!!!!)
"fixed" now 😛